lib/Interpreter/Compatibility.h

```diff
@@ -183,7 +183,6 @@ inline void codeComplete(std::vector<std::string>& Results,
 
 #endif // USE_CLING
 
-#ifdef USE_REPL
```

Comment:
These checks should be turned into `#ifndef USE_CLING`.

Reply:
You should find all such cases and rework them just like this one.

Reply:
Sure. Thanks !

Reply:
I have followed this norm for all definitions now.